cpa/parsing.go

```diff
@@ -139,3 +141,36 @@ func (err MultiError) Error() string {
 		return fmt.Sprintf("%d error(s) occurred: %s", len(err), strings.Join(messages, "; "))
 	}
 }
+
+//LoadPolicyFile takes policy file path as an input, and returns parsed policy
+func LoadPolicyFile(filePath string) (*Policy, error) {
+	documentBundle := map[string]string{}
+	fileContent, err := ioutil.ReadFile(filepath.Clean(filePath))
+	if err != nil {
+		return nil, fmt.Errorf("failed to read file: %w", err)
+	}
+	documentBundle[filepath.Base(filePath)] = string(fileContent)
+	return ParseBundle(documentBundle)
+}
+
+//LoadPolicyDirectory takes path of directory containing policies as an input, and returns parsed policy
+//every file in the top-level of the directory (non-recursive) will be considered as a policy file for parsing
+func LoadPolicyDirectory(directoryPath string) (*Policy, error) {
+	documentBundle := map[string]string{}
```

Comment:
initialise bundle after reading directory and initialize with length of files

Reply:
is required